llvm/lib/CodeGen/AsmPrinter/DwarfUnit.cpp

```diff
@@ -83,11 +83,13 @@ DwarfUnit::DwarfUnit(dwarf::Tag UnitTag, const DICompileUnit *Node,
                      AsmPrinter *A, DwarfDebug *DW, DwarfFile *DWU)
     : DIEUnit(UnitTag), CUNode(Node), Asm(A), DD(DW), DU(DWU) {}
 
+unsigned DwarfTypeUnit::UniqueID = 0;
```

Comment:
global state like this isn't workable in LLVM as it may be used as a library - we'll need some other way to track this numbering